### ISSUE TITLE
Report this content

### ISSUE THREAD

**Issue by prk-dismoi** [dis-moi/mobile_poc#215]
A feature to enable users to report content. Initially a store compliance requirement, will extend to include a quality feedback mechanism.

**Comment by prk-dismoi** [comment]
@MaartenLMEM @JalilArfaoui @felix-lambert your feedback welcome on PART 1 i.e. how the reporting tool should look like on the contribution screen UI.

![image](https://user-images.githubusercontent.com/80406294/128013927-bdacfc2a-63db-4018-b4e9-af83e3aaaa8b.png)

**Comment by MaartenLMEM** [comment]
> Also worried version 1 makes it a but confusing i.e. is it the bin that is used to report?

Right it's confusing. 
For option 1, the way to make it work is putting trash and heart on left side, and "signaler un problème" alone on the right side.

**Comment by prk-dismoi** [comment]
![image](https://user-images.githubusercontent.com/80406294/128023992-fc228d58-1e10-4142-8452-9153781ff392.png)

1. heart left, bin left, report right  - problem here is that large "like" numbers will push the bin towards the middle
2. heart left, bin middle, report right - personally i don't like that bin as a center piece
3. bin right, heart middle, report right - high number will clash with the reporting link

**Comment by prk-dismoi** [comment]
![image](https://user-images.githubusercontent.com/80406294/128025600-8fed76a3-0361-4b72-952d-77b413541edd.png)

Still my favourite. good stuff on the right, bad stuff on the left, no highlighted (center) item.

**Comment by JalilArfaoui** [comment]
> ![image](https://user-images.githubusercontent.com/80406294/128025600-8fed76a3-0361-4b72-952d-77b413541edd.png)
> 
> Still my favourite. good stuff on the right, bad stuff on the left, no highlighted (center) item, and no translation required.

Like it too … with text on hover (mouse) or long press (touch screen)

**Comment by MaartenLMEM** [comment]
1st solution ![image](https://user-images.githubusercontent.com/7824485/128037143-2541f1e0-548e-4043-baa4-5cd6f25edab4.png) seems me the most suitable solution for now.
I would just rename it `"Signaler un problème"`. Because `"signaler ce contenu`" looks a kind of legal report.

I prefer it to your favourite @prk-dismoi for two reasons :
- By flag, I think most of people will understand "report something not following the rules", "dénoncer un contenu inapproprié". 
For example, as user if I find the message to far from the product I am looking, I will just trash it, I won"t _flag it_.
If I the UI is dsiplayed on my screen out of chrome, I ll be less likely to report it with a flag than with "signaler un problème" label.
- Icons near one each to other can lead to missclick

**Comment by prk-dismoi** [comment]
As a user I dont see your concerns as a problem. Personally if I want to help I would naturally use the flag, but maybe it's an english thing.
With the sentence I also feel we're putting the focus too prominently on negativity. Problem is not a happy word, feels like we're unsure and that problems could be looming around every corner. 
Emphasising the feedback gathering aspect of it is a bit detrimental to the UI, so if we decide getting feedback on mismatch is of primary importance here, then we can go for 1. If not I still find my fav more fitting (we can work out the spacing).

**Comment by prk-dismoi** [comment]
![image](https://user-images.githubusercontent.com/80406294/128049931-1a104926-7416-4e70-b185-015e28a3848d.png)

Feedback welcome on this one. Wording, etc...

**Comment by MaartenLMEM** [comment]
> As a user I dont see your concerns as a problem. Personally if I want to help I would naturally use the flag, but maybe it's an english thing.

There is a difference. But can"t say if it is consequent, thus if it is worth regarding default on user side you underlined : negativity.

> With the sentence I also feel we're putting the focus too prominently on negativity. Problem is not a happy word, feels like we're unsure and that problems could be looming around every corner.

I agree about this risk. Hard to measure, but a real fact for people who will see it.

> Emphasising the feedback gathering aspect of it is a bit detrimental to the UI, so if we decide getting feedback on mismatch is of primary importance here, then we can go for 1. If not I still find my fav more fitting (we can work out the spacing).

Thanks for pointing out these stakes. Let's go on ideal UI for user, I mean with a flag. 
I we need more feedback at a some time or another, we ll have the label "problem" option.

**Comment by MaartenLMEM** [comment]
About reasons of flagging, I would not mention "**arnaque**". I suppose it can exist, that is not good for trust. Also people choose who they follow. We are not here to tell some contributor to shut up because a consumer finds it not worth. 

To sum up and add some new ideas : 

`Cette contribution 
- est sans rapport avec la page visitée
- ne m'intéresse pas
- est du spam
- me semble illégale
- autre

Remarques : ____

Mail : ________
*L'application est anonyme, c'est notre seul moyen de vous contacter pour en savoir plus.
 
[  ] Me désabonner du contributeur
[Valider]

**Comment by prk-dismoi** [comment]
Remarques : ____
Mail : ________
*L'application est anonyme, c'est notre seul moyen de vous contacter pour en savoir plus.

@MaartenLMEM Perso as a random user (but thats only me) I would never add notes here, and would still question why you want my email. "I told you what I had to say with the radio buttons, and dont have time/will to further help you improve your app, its not that important to me, I'm just browsing the web on my phone while waiting for the bus" (random use case). 

We might also need full screen for this UI. Not saying it should not be done, but for MVP compliance I would just recommend the 4 radios only? I will try to mock it up nevertheless so we can see how it could look.

**Comment by MaartenLMEM** [comment]
I agree, not sure at all mail field is a good idea, and obviously not MVP. Yes, keep it simple, easier to handle technically also.

**Comment by prk-dismoi** [comment]
![image](https://user-images.githubusercontent.com/80406294/128515555-601e4afc-fa91-444d-a9bb-c1a358ca0018.png)

**Comment by prk-dismoi** [comment]
Spoke with @MaartenLMEM. We're both good with the light (left) version initially. Will Zeplin it shortly.

**Comment by MaartenLMEM** [comment]
Derniers détails : 

- **Cette contribution** => Cette contribution...

- illégal~~e~~

**Comment by prk-dismoi** [comment]
> Derniers détails :
> 
> * **Cette contribution** => Cette contribution...
> * illégal~e~

@MaartenLMEM 
1. you mean text not bold?
2. I dont understand what you mean

**Comment by MaartenLMEM** [comment]
1. Yes text not bold and dot dot dot (but it s a detail and I am not sure of the result)
2. ok my bad : illegale vith e since it s linked to contribution
